getting-started/how-to-run-a-local-network/decentralizing-a-local-network.md

```diff
@@ -6,6 +6,43 @@ If your goal is to have a local network that closely resembles the public ICON c
 2. The last main validator's (pRep's) power should be more than `totalSupply * 0.2%` in power descending order.
 3. Revision >= 6
 
+The amount of registered validators and `totalSupply` can be obtained with the following RPC JSON calls.
```

Comment:
Define the command line instruction using goloop or curl:

`goloop rpc <json_file>` or whatever

Reply:
done